cammac60/MacRae-Photo
# Dir setup

### What does this PR do? 

- Sets up `/images` directory in public w/ a dir for photos (to be hidden), a dir for icons, and several other random photos for backgrounds and such.
- Adds a `/components` directory with child dirs for each component (and files for each).
- Updates manifest and `index.html` to change document title, icons, and remove un-needed create-react-app code.
- Adds files for API calls.
- Adds SASS vars file. 
- Adds data file for image data (possibly to be moved to the server). 
- Sets up basic structure in `App.js`

----------

### How to manually test: 

- `git fetch`
- `git checkout dir-setup`
- `npm start`
- Ensure that the app starts on localhost w/o errors. 
- Ensure that the document title and logo are set correctly. 

----------

### Tickets: 

- Resolves #3 
- Resolves #4 
- Resolves #5
